.github/workflows/s3_upload.yml

```diff
@@ -32,24 +32,15 @@ jobs:
         run: |
           npm run build
 
+      - name: Install AWS CLI
+        run: |
+          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install
+          aws --version
```

Comment:
Is there a way to install the aws cli via the package manager instead? Do we expect sudo to work here?

Reply:
I've changed the script to use `pip`